docs/explanation/rocks.rst

```diff
@@ -3,22 +3,62 @@
 ROCKs
 =====
 
-ROCKs are container images that are compliant with the `Open Container
-Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
-This means that ROCKs are interoperable with other tools that work with
-OCI-compliant images.
+ROCKs are container images based on Ubuntu designed to meet
+cloud-native software's security, stability, and reliability requirements.
+
+ROCKs were created to address two primary needs. Firstly, it tackles
+the current issues with container image security, as many popular
+container images often have known vulnerabilities.
+Secondly, it aims to provide a smaller container image based on
+Ubuntu to reduce the resource footprint on the host.
```

Comment:
This is controversial. 

> Firstly, it tackles the current issues with container image security, as many popular container images often have known vulnerabilities.

How?

---

> it aims to provide a smaller container image based on Ubuntu to reduce the resource footprint on the host.

This isn't true. Alpine and distroless images are still smaller.

Reply:
For the first one, by building on top of Ubuntu but also thanks to Chisel.

For the second, I didn't say that ROCK image is smaller than Alpine or Distroless; what gives you this impression?

Reply:
> For the first one, by building on top of Ubuntu but also thanks to Chisel.

but Ubuntu isn't the most secure distribution image out there.

> For the second, I didn't say that ROCK image is smaller than Alpine or Distroless; what gives you this impression?

well your statement claimed that "the goal is to make smaller images", but when you think about it: an alpine-based application image will be smaller than the equivalent ROCK. And a distroless image will also be smaller than the equivalent Chiselled ROCK.

Reply:
> but Ubuntu isn't the most secure distribution image out there.

Yeah, but it's one of the most secure. Maybe it's the choose of words. I wanted to said that the container will be more secure. 

> well your statement claimed that "the goal is to make smaller images", but when you think about it: an alpine-based application image will be smaller than the equivalent ROCK. And a distroless image will also be smaller than the equivalent Chiselled ROCK.

Yeah, you are right, I think it's a problem of words. I wanted to say that ROCK will give you a smaller image than the Ubuntu one, but not smaller than Alpine or Distroless.

Have some ideas about how I can rephrase these sentences?

Reply:
I don't think these are the real values. if you check our ongoing ROCKs@Canonical whitepaper, there are plenty of values, ones stronger than others. 

IMHO, saying that "we are one of the most secure" or "amongst the smaller" is the same as saying that we are aiming to do something that others do better.

For me, the real value and innovation behind Rockcraft is:
 - the declarative format (yes there are other tools too, but unlike security and size, UX is a subjective topic so we can aim to be the best at it)
 - the Ubuntu experience - we eat our own dog food, cause we believe in ubuntu and no one really offer something that can build a truly distro-based distroless image :) this paradox is great, but we actually allow this -> chiselled ROCKs are Ubuntu distroless images :D we broke the convention
 - the consistency - while disregarded by many, one of the problems out there is there every docker image has its own ENTRYPOINT and CMD, so everytime you need to use a new image, to need to get acquainted with it. and the only way to do it is to either read the docs, or inspect the inner contents of the image. In ROCKs, once you get used to Pebble, you get used to every ROCK out there and you'll only need to know the application-specific options

(among others)

Reply:
Thanks, i have updated this section; tell me if you find this better

Reply:
I think this is where the lines are blurred between ROCKs and Rockcraft. These list items are very similar to those in the "Why use Rockcraft?" document.

What we need at the start of this document (rocks.rst) is a description of what ROCKs are. Something like this:
> ROCKs are OCI-compliant container images, built on an Ubuntu base, with a consistent entry point.

There is a certain amount of unpacking that needs to happen with that sentence, which can be done in later sentences/paragraphs.
We can also say why it is important that a ROCK has those properties. Some of this is already present in the text:

> ROCKs comply with the [Open Container Initiative](https://opencontainers.org/) (OCI) [image format specification](https://github.com/opencontainers/image-spec/blob/main/spec.md). ROCKs can be stored in container registries (e.g. DockerHub, ECR, ACR,..) and used with any OCI-Compliant tools (e.g. Docker, Podman, Kubernetes,…).

We can then get into the details of what a ROCK contains and how one is built. Perhaps the details of how one is built could be moved into another document later.